assets/css/style.css

```
@@ -201,3 +201,80 @@ body, html, ul, ol, li, p, h1, h2, h3, h4 {
 .img-section{
   margin-bottom: 70px;
 }
+
+.menu-section {
+  background-image: url(../css/images/menu-bg.png);
+  display: flex;
+  justify-content: space-evenly;
+  padding: 50px 0 100px 0;
+}
+
+.category-header {
+  text-align: center;
+}
+
+.dish-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.dish-category {
+  margin-top: 50px;
+  width: 420px;
+}
+
+.category {
+  color: #cc9900;
+  font-family: "YesevaOne", 'Arial', 'Times New Roman', sans-serif;
+  font-size: 35px;
+  font-weight: normal;
+  margin-bottom: 5px;
+}
+
+.dish-name {
+  color: #cc9900;
+  font-family: "YesevaOne", 'Arial', 'Times New Roman', sans-serif;
+  font-size: 19px;
+  font-weight: lighter;
+  padding-bottom: 5px;
+}
+
+.dish-price {
+  color: #cc9900;
```

Same here and below..
```suggestion
  color: #c90;
```

---

fixed